src/envs/pybullet_robots.py

```diff
@@ -307,3 +306,105 @@ def create_single_arm_pybullet_robot(
                                   finger_action_tol, max_vel_norm,
                                   physics_client_id)
     raise NotImplementedError(f"Unrecognized robot name: {robot_name}.")
+
```

Comment:
all this is just copied over from pybullet_utils (now deleted), but PyBulletState was moved into pybullet_env.py